src/components/AccountDisplay.tsx

```diff
@@ -79,9 +81,9 @@ const AccountDisplay = ({ address = '', addressKind, balance, friendlyName, hide
 
   return (
     <Container onClick={onClick} className={classes.container}>
-      <div className={classes.icon}>{address && <AccountIdenticon address={address} />}</div>
-      <div className={classes.address}>
-        <p>{displayText()}</p>
+      <div className={classes.icon}>{<AccountIdenticon address={address} />}</div>
```

Comment:
I guess now it's not conditional we can actually remove the `'init'` stuff, no?

Reply:
Yeap.. i did not? FFS!!!!